cpp/velox/jni/jni_wrapper.cc

```diff
@@ -47,14 +50,16 @@ jint JNI_OnLoad(JavaVM* vm, void* reserved) {
 void JNI_OnUnload(JavaVM* vm, void* reserved) {
   JNIEnv* env;
   vm->GetEnv(reinterpret_cast<void**>(&env), JNI_VERSION);
+  veloxPool_.reset();
```

Comment:
If we release the pool in `OnUnload` should we create it in `OnLoad`?

Reply:
Fixed by creating it in OnLoad.